torch/legacy/nn/MultiMarginCriterion.py

```diff
@@ -4,7 +4,7 @@
 
 class MultiMarginCriterion(Criterion):
 
-    def __init__(self, p=1, weights=None, margin=1, sizeAverage=True):
+    def __init__(self, p=1, margin=1, weights=None, sizeAverage=True):
```

Comment:
This makes it more consistent, but it is a breaking change. It's not suggested to use legacy modules, but let's not break it.